.gitignore

```diff
@@ -215,4 +215,5 @@ compile_commands.json
 build/
 
 # Idea based stuff
-.idea/
\ No newline at end of file
+.idea/
+.vscode/
```

Comment:
```suggestion

# VS Code
.vscode/
```
This makes it look like it's a part of "Idea based stuff".

Reply:
:+1: